style.css

```diff
@@ -753,4 +753,63 @@ hr {
   .project-detail-contener {
     padding: 24px;
   }

```

Comment:
- [ ] So far you have a good design 👏 . However, we need to do some improvements. I suggest that you ensure that the languages list is vertically centered in all the desktop screen sizes. In addition, I suggest that the width of the card is responsive to all the screen sizes. If you notice, at `772px` the contents of the card cross the borders of the card. This reduces the UI interface of the application. Therefore I request to apply these changes to ensure that the UI of your portfolio is improved.

Your design 😎 
![image](https://github.com/patrick-amponsah/Portfolio-Project/assets/99394374/dda71640-caca-48b0-a5fa-169e5318bc1b)

Original design 😎 
![image](https://github.com/patrick-amponsah/Portfolio-Project/assets/99394374/c651a475-9e3f-4a96-8093-a60d45a85535)